### ISSUE TITLE
[Documentation] Change the colours of the code section 

### ISSUE THREAD

**Issue by kud** [mjmlio/mjml#1696]
The code on the documention is tough to read.
Maybe it's because I'm a bit colourblind but it hurts my eyes to read the code.

![image](https://user-images.githubusercontent.com/655838/63956739-4f385b80-ca87-11e9-9908-8fcd103c5d9b.png)

Some tools:

- https://accessible-colors.com/
- https://webaim.org/resources/contrastchecker/

https://codemirror.net/demo/theme.html#material could be nice.

Thank you!

**Comment by kud** [comment]
Code's here: https://github.com/mjmlio/slate/tree/master/source/stylesheets

**Comment by dougfabris** [comment]
Hey @kud I agree with you, the code colors of documentation hurts our eyes. So I tried this combination without make a big change in actual color palette. First of all, I believe we need to change background-color and his shadows and then, change the red color to another that make more contrast and less brightness. Do you think it could be better like that?

![code](https://user-images.githubusercontent.com/27704687/64142590-fba97300-cde2-11e9-9750-764dcdba3640.png)
![contrast](https://user-images.githubusercontent.com/27704687/64142591-fba97300-cde2-11e9-8743-7dd398613fe7.png)
![contrast-ratio](https://user-images.githubusercontent.com/27704687/64142592-fc420980-cde2-11e9-9fad-c13974f69513.png)

**Comment by kud** [comment]
That's really great! Thank you to answer so quickly too!

I'm more than ok with those colours! Good for me.

**Comment by dougfabris** [comment]
@kud You're very welcome! The MJML project helps me a lot... I'm always trying to do the same, as I can.

**Comment by iRyusa** [comment]
cc @ngarnier

**Comment by iRyusa** [comment]
@kud @dougfabris 

Is current try it live (https://mjml.io/try-it-live) theme accessible enough ? So we can maybe use this one instead ? Or we should update it to so we can re-use the same on both doc & try it live ?

**Comment by kud** [comment]
Yes, it's ok for me, with maybe a darker background.

**Comment by dougfabris** [comment]
@iRyusa I agree with @kud. To be more accessible, maybe we will have to use a darker background, to get a nice contrast with the red color. This tool is a good option to check the contrast ratio: https://contrast-ratio.com/
